### PR TITLE
Adds new snacks to vending machines!

### PR DESCRIPTION
**Pull request by thexenomorph21**
3 Candy, 6 cans!

## What Does This PR Do

This PR adds 5 new cans to the game and 1 new candy item! The cans in no particular order are Voobra, Blue Cow, Solar Espresso, K&N/A Root Beer and Diet Space Cola. The 1 candy item added is a biscuit bar. These are just general snack items, and function pretty much identically to their counterparts. The only real difference is with the Solar Espresso and the Blue Cow, both of these increase your heart rate!

## Why It's Good For The Game

I believe these additions are good for the game as they add more variety to roleplay and food stuffs! While the vending machines already have variety, I sometimes feel like they're lacking in choice. _What do you MEAN Diet Space Cola isn't on board the Cyberiad?! I'm on a diet!_

## Images of changes
image

## Testing
Threw the drinks, made them trash, and, well, drunk them.

## Changelog
:cl: thexenomorph21

add: added Voobra, Blue Cow, Solar Espresso, K&N/A Root Beer, Diet Space Cola, and Nebula Fizz as drinks available in vendors. 
add: added Biscuit Bar and Plasma Puffs as snacks available in vendors.
add: added Stargums as a snack available in vendors.

/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/drinks_reagents.dm [hunk 1]
```diff
@@ -737,3 +750,53 @@
...
+	reagent_state = LIQUID
+	color = "#100800" // rgb: 16, 8, 0
+	adj_drowsy = -10 SECONDS
+	heart_rate_increase = 10
+	drink_icon = "lemonglass"
+	drink_name = "Glass of Blue Cow!"
+	drink_desc = "Cow. . . cow what?"
+	taste_description = "the fact that your insides are burning."
+
+/datum/reagent/consumable/drink/espresso
```
Comment (Fox-McCloud) [inline code/modules/reagents/chemistry/reagents/drinks_reagents.dm:791]: Why isn't this a subtype of coffee?

Comment (thexenomorph21) [reply code/modules/reagents/chemistry/reagents/drinks_reagents.dm:791]: because I'm clearly a little stupid, I can update it, if that works?

Comment (thexenomorph21) [reply code/modules/reagents/chemistry/reagents/drinks_reagents.dm]: > Why isn't this a subtype of coffee?

It is now!

**Comment by ATP-Engineer** [comment]
Did you make the sodas/energy drinks IED item compatible?

**Comment by thexenomorph21** [comment]
> Did you make the sodas/energy drinks IED item compatible?

They should already be IED item compatible due to them being a subtype of can drinks, I think!

**Comment by farie82** [comment]
Heyo, welcome to the codebase!
Can you change your CL (changelog) to be more descriptive? Like:
:cl:
add: added Voobra, Blue Cow, Solar Espresso, K&N/A Root Beer and Diet Space Cola as drinks available in vendors
add: ...
/:cl:

This should also be the format for a CL. No dates or double CLs are required. That will be done automatically when the PR is merged

**Comment by thexenomorph21** [comment]
> Heyo, welcome to the codebase! Can you change your CL (changelog) to be more descriptive? Like: 🆑 add: added Voobra, Blue Cow, Solar Espresso, K&N/A Root Beer and Diet Space Cola as drinks available in vendors add: ... /🆑
> 
> This should also be the format for a CL. No dates or double CLs are required. That will be done automatically when the PR is merged

Thank you for the clarification, everything should be in order now!

**Comment by farie82** [comment]
> Thank you for the clarification, everything should be in order now!

Nearly correct. You can leave out the enters between the tags.
```
:cl:
add: added Voobra, Blue Cow, Solar Espresso, K&N/A Root Beer, Diet Space Cola, and Nebula Fizz as drinks available in vendors.
add: added Biscuit Bar and Plasma Puffs as snacks available in vendors.
add: added Stargums as a snack available in vendors.
/:cl:
```
As an extra. You can leave out the author name if it is the same as the author of the PR on github.

File: code/modules/food_and_drinks/food/foods/candy.dm [hunk 2]
```diff
@@ -124,6 +124,17 @@
...
+/obj/item/reagent_containers/food/snacks/candy/plasmapuffs
+	name = "plasma puffs"
+	desc = "A savory snack made from a mixture of dehydrated plasma and various spices!"
+	icon_state = "plasmapuffs"
+	trash = /obj/item/trash/plasmapuffs
+	filling_color = "#FFFFFF"
+	bitesize = 4
+	list_reagents = list("plasma" = 0.1, "sugar" = 20)
+	tastes = list("plasma" = 1)
+
```
Comment (lewcc) [inline code/modules/food_and_drinks/food/foods/candy.dm:136]: Sorry for these pester reviews, just for consistency with the rest of the file. If you wouldn't mind removing the extra spaces around your edits in the rest of the file it'd be nice
```suggestion
```

File: code/modules/reagents/chemistry/reagents/drinks_reagents.dm [hunk 3]
```diff
@@ -324,6 +324,21 @@
 	drink_desc = "A nice and refrshing beverage while you are reading."
 	taste_description = "milkish coffee"
 
+
```
Comment (lewcc) [inline code/modules/reagents/chemistry/reagents/drinks_reagents.dm:327]:
```suggestion
```

File: code/modules/food_and_drinks/food/foods/candy.dm [hunk 4]
```diff
@@ -136,6 +147,18 @@
 	list_reagents = list("nutriment" = 1, "chocolate" = 1)
 	tastes = list("chocolate" = 1)
 
+/obj/item/reagent_containers/food/snacks/candy/biscuitbar
+	name = "Biscuit bar"
+	desc = "A chocolate candybar, with bits of crunchy biscuit inside!."
```
Comment (lewcc) [inline code/modules/food_and_drinks/food/foods/candy.dm:152]:
```suggestion
	desc = "A chocolate candybar, with bits of crunchy biscuit inside!"
```

File: code/game/objects/items/trash.dm [hunk 5]
```diff
@@ -110,6 +110,19 @@
...
+
+/obj/item/trash/plasmapuffs
+	name = "plasma puffs"
+	icon_state = "plasmapuffs"
+
+/obj/item/trash/stargums
+	name = "stargums"
+	icon_state = "stargums"
+
+
```
Comment (lewcc) [inline code/game/objects/items/trash.dm:125]:
```suggestion
```

**Comment by SteelSlayer** [comment]
Once this is deconflicted and the reviews are addressed, it can be merged.

**Comment by farie82** [comment]
Closing for inactivity

**Pull request closed without merging**